styles/style.css

```diff
@@ -6,3 +6,101 @@ Green: #00B172
 White: #FFF
 
 */
+* {
+    font-family: "Verdana";
+}
+
+.spotify-logo{
+    width: 120px;
+    margin-right: 1rem;
+    display: flex;
+    align-items: flex-start;
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    margin: 2%;
+    align-items: center;
+}
+header >div {
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+}
+
+ul { 
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+  }
+li{
+    margin-right: 20px;
+}
+
+.container1{
+    display: flex;
+    justify-content:center;
+    align-items:center;
+    flex-direction: column;
+    position: relative;
+    height: 800px;
+    background-size: cover;
+    background-image: url(https://raw.githubusercontent.com/ironhack-labs/lab-css-spotify-clone/master/images/landing.jpg);
+}
+p{
+    color: white;
+    font-size: 1.5em;
+    text-align: center;
+ 
+}
+h1{
+    color: white;
+    font-size: 4em;
+ }
+
+h2 {
+    font-size: 30px;
+    text-align: center;
+    padding: 5px;
+}
+   
+
+h3 {
+    color:#00B172;
+    font-weight: bold;
+    //text-align: center;
+    font-size: 24px;
+}
+
+h4 {
```

Comment:
There is a lot of space between your text under "What's on Spotify". The reason for this is because the h3 and h4 that you are using have a lot of default margin. To reduce this space you can say:
h3, h4 {
 margin: 0
}